src/config.json

```diff
@@ -73,6 +73,15 @@
       "auxPalette": "reds",
       "colorReadsByMappingQuality": false
     }
+  },
+
+  "fileTypeToExtensions": {
+    "graph": ".xg,.vg,.hg,.gbz,.pg",
+    "haplotype": ".gbwt,.gbz",
+    "read": ".gam"
+  },
+
+  "MAXUPLOADSIZE": 5242880,
+  "pickerTypeOptions": ["mounted", "upload"]
```

Comment:
What is this for? Is it meant to let you turn off the upload feature? Because it won't on the server side.